migrate/20240701_load_balancer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:lb_algorithm, %w[round_robin source_hash])
```

Comment:
this might be fine and all, I know there was some mailing list discussion, do we have a sense of what we'd put in the manual, or is this more an internal experimentation jig, or whatever? Not a blocker, it's not like a new lb algorithm is going to show up even once every two...four...six...years in nftables world, so agility to try them out is justifiable.

Reply:
I was thinking in near future we start experimenting with least_conn.